src/lib/stringIndices.js

```diff
@@ -0,0 +1,16 @@
+  let stringIndices = (userString, userNumber) => {
+    let newArr = []
```

Comment:
newArr is redefined in a different way in each level of the if statements so remove this line, it is unnecessary.